features/daily/user_password.feature

```diff
@@ -0,0 +1,8 @@
+Feature: User password
+
+  Scenario: Change password using a user token
+    Given there are authentication users with info
+    | firstname | username | password |
+    | Greg      | greg     | 123test  |
```

Comment:
You can see [styleguide](https://github.com/wazo-pbx/wazo-acceptance/blob/master/STYLEGUIDE.md) for indent

Reply:
Because it's a missing ACL in the integration between the wazo-platforme and wazo-auth